Box.V2/Models/BoxUser.cs

```diff
@@ -24,7 +24,7 @@ public class BoxUser : BoxEntity
         public const string FieldJobTitle = "job_title";
         public const string FieldPhone = "phone";
         public const string FieldAddress = "address";
-        public const string FieldAvatarArl = "vatar_url";
+        public const string FieldAvatarUrl = "avatar_url";
```

Comment:
I think we should update it, but not sure the impact since this is public @cburnette